currency-converter/currency.css

```diff
@@ -65,25 +112,259 @@ input[type="number"] {
   border: 1px solid #ccc;
   border-radius: 4px;
   font-size: 1em;
+  z-index: 1;
 }
 
-button {
+.button {
   padding: 10px;
-  background: linear-gradient(rgb(3, 3, 3), rgb(52, 51, 51), rgb(4, 4, 4));
-  color: #ffffff;
+  background: linear-gradient(
+    rgb(3, 122, 62),
+    rgb(57, 180, 108),
+    rgb(3, 122, 62)
+  );
+  color: #0d0000;
   border: none;
   border-radius: 4px;
   font-size: 1em;
   cursor: pointer;
   transition: background-color 0.3s ease;
+  z-index: 1;
+}
+
+.button:active {
+  background-color: #3e8e41;
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
 }
 
 .conversionResult {
   font-size: 1.2em;
   font-weight: bold;
   margin-top: 15px;
-  color: #333;
+  color: #090101;
+  text-align: center;
+}
+
+.rates-grid {
+  margin-top: 10px;
+  z-index: 1;
+  display: flex;
+  justify-content: center;
+}
+
+#ratesTable {
+  width: 70%;
+  border-collapse: collapse;
+  z-index: 1;
+}
+
+#ratesTable th,
+#ratesTable td {
+  border: 1px solid #00010c;
+  padding: 8px;
   text-align: center;
+  z-index: 1;
+}
+
+#ratesTable th {
+  background-color: rgba(35, 130, 220, 0.8);
+  font-weight: bold;
+  z-index: 1;
+}
+
+.squares {
```

Comment:
I would name this class `.flags` because that it's what it contains.

Reply:
i have renamed the class to flags